ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -454,7 +454,12 @@ module.exports = Marionette.LayoutView.extend({
       .get('preferences')
       .get('resultFilter')
     if (resultFilter) {
-      this.handleFilter(CQLUtils.transformCQLToFilter(resultFilter), '#c89600')
+      this.handleFilter(
```

Comment:
:question: Can you do this instead?
`this.handleFilter(cql.read(resultFilter), '#c89600')`

Reply:
Do not believe so because resultFilter is now a filter tree object.

Reply:
If resultFilter is a filterTree, can do this instead? 
`this.handleFilter(resultFilter)`

I don't know how the CQLUtils methods work, but I would expect 
`CQLUtils.transformCQLToFilter(CQLUtils.transformFilterToCQL(resultFilter))`
to return resultFIlter (or a duplicate of it)

Reply:
You would think, but filter -> cql -> filter yields a different filter tree. More specifically, the value property on the return object is different. In its original state the map parser will throw an error - this seemed like a better solution than trying to rewrite the query -> to filter serialization.